packages/salesforcedx-vscode-core/src/commands/util/commandLogEntry.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export type CommandLogEntry = {
+  commandId: string;
+  timestamp: number;
+  duration: number;
+  exitCode?: number;
+  error?: string;
+  data?: any;
```

Comment:
unknown would be better for consuming the data in a type safe way.

Reply:
👍